app/models/project.rb

```diff
@@ -53,4 +55,11 @@ def member?(user)
   def classifiable_requests
     info_requests.where(awaiting_description: true)
   end
+
+  def extractable_requests
+    info_requests.where(
```

Comment:
We'll want to update this based on https://github.com/mysociety/alaveteli/pull/5654 – probably need extractable requests that _you_ have not submitted data to